umbral-pre/src/pre.rs

```diff
@@ -41,6 +42,47 @@ pub fn decrypt_original(
     dem.decrypt(ciphertext, &capsule.to_array())
 }
 
+/// Creates `num_kfrags` fragments of `delegating_sk`,
+/// which will be possible to reencrypt to allow the creator of `receiving_pk`
+/// decrypt the ciphertext encrypted with `delegating_sk`.
+///
+/// `threshold` sets the number of fragments necessary for decryption
+/// (that is, fragments created with `threshold > num_frags` will be useless).
```

Comment:
WDYT about returning an error on `threshold > num_frags`? Is there any reason we should permit creating "useless" fragments?

Reply:
Yes, it's already done in PyUmbral. I think originally I didn't do it because we'd have to return a `Result`, and it seemed like a very minor thing to worry about - after all, it is not an error, we're just doing exactly what the user asked for.

This can also be solved purely by type means - instead of `threshold` and `num_kfrags`, we can ask for `threshold` and `overhead` (or whatever the term might be), both `usize`. This way it is impossible to create less than `threshold` frags.

Reply:
Not in this PR, filed #48